eng/init-source-only.proj

```diff
@@ -33,6 +34,40 @@
           WorkingDirectory="$(PreviouslySourceBuiltPackagesPath)" />
   </Target>
 
+  <ItemGroup>
+    <SharedComponentsPrereqsArchivePath
+      Include="$(ExternalSharedComponentsTarballsDir)$(SourceBuiltArtifactsTarballName).*$(ArchiveExtension)" />
+  </ItemGroup>
+
+  <!--
+    Unpacks the previously source built archive produced from the 1xx feature band.
+    There are two possible sources for this archive:
+    1. The archive is specified by the user via the with-shared-components option in the build script.
+       This is what distro maintainers would typically use. This maps to the PreviouslySourceBuiltSharedComponentsArchivePath
+       property. This source takes precedence.
+    2. The archive is contained in the prereqs/packages/archive-shared-components directory.
+       This is what the prep script does by default. This provides a seamless experience for devs working in
+       the VMR, not requiring them to pass extra parameters by default when working in a non-1xx branch.
+       This scenario maps to the SharedComponentsPrereqsArchivePath item.
+  -->
+  <Target Name="UnpackSourceBuiltSharedComponentsArtifactsArchive"
+          Condition="'$(PreviouslySourceBuiltSharedComponentsArchivePath)' != '' or @(SharedComponentsPrereqsArchivePath) != ''">
```

Comment:
This model appears different than the source-built-artifacts.  With the PSB, the --with-packages is a directory where is PSB tarball is extracted.  This difference feels like a potential source of confusion.  Can we unify how these are handled?